File: model/Plan.cpp
//
// Created by Lucas Kujawski on 26/12/2020.
//

#include "Plan.h"

int Plan::getPricePerMonth() const {
    return price_per_month;
}

void Plan::setPricePerMonth(int pricePerMonth) {
    price_per_month = pricePerMonth;
}

int Plan::getAllowedItemsPerMonth() const {
    return allowed_items_per_month;
}

void Plan::setAllowedItemsPerMonth(int allowedItemsPerMonth) {
    allowed_items_per_month = allowedItemsPerMonth;
}

File: model/Plan.py
from sqlalchemy.ext.declarative import declarative_base
from sqlalchemy import Column, Integer
from sqlalchemy import ForeignKey

Base = declarative_base()


class Plan(Base):
    __tablename__ = 'plan'
    id = Column(Integer, primary_key=True)
    price_per_month = Column(Integer)
    allowed_items_per_month = Column(Integer)
    client_id = Column(Integer, ForeignKey('client.id'))